lib/image_optim/worker/jhead.rb

```diff
@@ -7,6 +7,8 @@ class Worker
     #
     # Jhead internally uses jpegtran which should be on path
     class Jhead < Worker
+      TIMEOUT_OPTION = timeout_option
```

Comment:
You are right. I understood only later that you've implemented the timeout in the way that if any worker timeouts, than optimisation will not continue. But if workers that timeout are just skipped, it is better to now allow jhead to timeout.